sysbuild/CMakeLists.txt

```diff
@@ -432,12 +432,6 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
       endif()
     endif()
 
-    if(SB_CONFIG_MCUBOOT_FPROTECT_ALLOW_COMBINED_REGIONS)
```

Comment:
this Kconfig sysbuild property need to be removed as not used anymore by this PR.